src/main/java/org/breedinginsight/brapi/v2/model/response/mappers/ListQueryMapper.java

```diff
@@ -0,0 +1,35 @@
+package org.breedinginsight.brapi.v2.model.response.mappers;
+
+import lombok.Getter;
+import org.brapi.v2.model.germ.BrAPIGermplasm;
+import org.breedinginsight.utilities.response.mappers.AbstractQueryMapper;
+
+import javax.inject.Singleton;
+import java.util.Map;
+import java.util.function.Function;
+
+@Getter
+@Singleton
+public class ListQueryMapper extends AbstractQueryMapper {
+
+    private Map<String, Function<BrAPIGermplasm, ?>> fields;
```

Comment:
I think `BrAPIGermplasm` should be removed from the type parameter, and the `ListQueryMapper` class be made more generic

Reply:
fixed copy and paste errors